scipy/optimize/zeros.py

```diff
@@ -192,31 +195,46 @@ class of similar problems can be solved together
     >>> root
     1.0000000000000016
 
-    Only ``fprime`` provided, use Newton Raphson method
+    Only ``fprime`` is provided, use the Newton-Raphson method:
 
     >>> root = optimize.newton(f, 1.5, fprime=lambda x: 3 * x**2)
     >>> root
     1.0
 
-    Both ``fprime2`` and ``fprime`` provided, use Halley's method
+    Both ``fprime2`` and ``fprime`` are provided, use Halley's method:
 
     >>> root = optimize.newton(f, 1.5, fprime=lambda x: 3 * x**2,
     ...                        fprime2=lambda x: 6 * x)
     >>> root
     1.0
 
-    A sequence of initial guesses and arguments is provided, use vectorized
-
-    >>> import numpy as np
```

Comment:
@rgommers sorry I missed that this line had been removed, I don't think the doctest will run correctly if numpy isn't imported, because it is called below on lines 217 and others.

Reply:
The removal is correct. In every example in scipy we don't have that line, it's implied. And we run doctests, if they fail then TravisCI will be red.